core-ui/src/components/Clusters/views/ClusterList.js

```diff
@@ -43,13 +43,13 @@ function ClusterList() {
   }
 
   const styleActiveCluster = entry => {
-    return entry.kubeconfig['current-context'] === activeClusterName
+    return entry?.kubeconfig['current-context'] === activeClusterName
```

Comment:
if the error is "cannot read kubeconfig of undefined", then "current-context" won't be defined as well 
```suggestion
    return entry?.kubeconfig?.['current-context'] === activeClusterName
```